app/src/main/java/org/gdg/frisbee/android/view/CircleTransform.java

```diff
@@ -0,0 +1,50 @@
+package org.gdg.frisbee.android.view;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapShader;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+
+import com.squareup.picasso.Transformation;
+
+/**
+ * Created by <a href="mailto:marcusandreog@gmail.com">Marcus Gabilheri</a>
+ *
+ * @author Marcus Gabilheri
+ * @version 1.0
+ * @since 7/26/15.
+ */
+public class CircleTransform implements Transformation {
```

Comment:
I am also talking about BitmapBorderTransformation. They are doing the same
job.

On Mon, Jul 27, 2015, 4:09 PM Marcus Gabilheri <notifications@github.com>
wrote:

> In app/src/main/java/org/gdg/frisbee/android/view/CircleTransform.java
> <https://github.com/gdg-x/frisbee/pull/421#discussion_r35533007>:
>
> > +
> > +import android.graphics.Bitmap;
> > +import android.graphics.BitmapShader;
> > +import android.graphics.Canvas;
> > +import android.graphics.Paint;
> > +
> > +import com.squareup.picasso.Transformation;
> > +
> > +/**
> > + * Created by <a href="mailto:marcusandreog@gmail.com">Marcus Gabilheri</a>
> > + *
> > + * @author Marcus Gabilheri
> > + * @version 1.0
> > + * @since 7/26/15.
> > + */
> > +public class CircleTransform implements Transformation {
>
> Hmmm. The only one that I could find was the BitmapBorderTransformation.
> is that in another branch?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/421/files#r35533007>.
>